File: src/main/java/de/qabel/core/exceptions/QblException.java
package de.qabel.core.exceptions;

public abstract class QblException extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = 4261199805687953191L;


	public QblException() {
	}

	public QblException(String msg) {
		super(msg);
	}
}

File: core/src/main/java/de/qabel/core/exceptions/QblException.kt
package de.qabel.core.exceptions

abstract class QblException : Exception {


    constructor() {
    }

    constructor(msg: String) : super(msg) {
    }

    constructor(e: Throwable) : super(e) {
    }

    constructor(msg: String, e: Throwable) : super(msg, e) {
    }

    companion object {

        /**

         */
        private val serialVersionUID = 4261199805687953191L
    }
}
